Bullwinkle.class.nut

```diff
@@ -442,6 +448,14 @@ class Bullwinkle {
                 }
             }
         }
+
+        // If packages still pending schedule next run
+        if ( _packages.len() > 0 ) {
+            _watchdogTimer = imp.wakeup(0.5, _watchdog.bindenv(this));
```

Comment:
A question not related to this particular change set. Does it make sense to have this timeout value configurable? Or at least to define it as a constant.